### PR TITLE
bugfix in using search_kwargs in Vectara retriever

### PR DESCRIPTION
**Pull request by ofermend**
- **Description:** fix a bug that prevented as_retriever() in Vectara to use the desired input arguments
  - **Issue:** as_retriever did not pass the arguments properly
  - **Tag maintainer:** @baskaryan
  - **Twitter handle:** @ofermend

### CODE REVIEW

**Comment by efriis** [comment]
Hey @ofermend ! Could you check your `ruff` version? Recently upgraded to `0.1.5`

File: libs/langchain/langchain/schema/vectorstore.py [hunk 1]
```diff
@@ -608,7 +608,7 @@ def as_retriever(self, **kwargs: Any) -> VectorStoreRetriever:
         """
         tags = kwargs.pop("tags", None) or []
         tags.extend(self._get_retriever_tags())
-
+
```
Comment (efriis) [inline libs/langchain/langchain/schema/vectorstore.py:611]:
```suggestion

```

Comment (ofermend) [reply libs/langchain/langchain/schema/vectorstore.py]: Yes, updated my ruff version and removed that extra space line.

Comment (efriis) [reply libs/langchain/langchain/schema/vectorstore.py]: 1 more formatting error I think - just a newline where those spaces were removed

Comment (efriis) [reply libs/langchain/langchain/schema/vectorstore.py]: If you run `make format` from libs/langchain, this should handle all these things automatically!

Comment (ofermend) [reply libs/langchain/langchain/schema/vectorstore.py]: Thanks. Fixed. (sorry was running "lint" from antoher folder and it prints all kinds of other unrelated things. Will know about make format going fwd)

Comment (efriis) [reply libs/langchain/langchain/schema/vectorstore.py]: No problem! Thanks for fixing.

**Review by efriis — APPROVED** [review]

File: libs/langchain/langchain/vectorstores/vectara.py [hunk 2]
```diff
@@ -451,12 +450,13 @@ class VectaraRetriever(VectorStoreRetriever):
     """Vectara vectorstore."""
     search_kwargs: dict = Field(
         default_factory=lambda: {
-            "lambda_val": 0.025,
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/vectara.py:454]: out of curiosity what's the motivation for this?

Comment (ofermend) [reply libs/langchain/langchain/vectorstores/vectara.py:454]: This controls hybrid search. It's not a massive difference, just thinking that users may not be aware that the default is some form of hybrid search.

**Pull request merged**